main/swadge2024.c

```diff
@@ -163,12 +165,19 @@ static swadgeMode_t* cSwadgeMode = &mainMenuMode;
 /// @brief A pending Swadge mode to use after a deep sleep
 static RTC_DATA_ATTR swadgeMode_t* pendingSwadgeMode = NULL;
 
+/// @brief A minimal Swadge mode to be displayed on top of others
+/// Note that it only gets init/exit and mainLoop callbacks.
```

Comment:
I have a slight preference for using `/** ... */` comments when they're on multiple lines. I'm being stylistically nitpicky, so I must be tired.